neurox/interpretation/utils.py

```diff
@@ -139,7 +143,14 @@ def count_target_words(tokens):
 
 
 def create_tensors(
-    tokens, activations, task_specific_tag, mappings=None, task_type="classification", binarized_tag = None, balance_data = False
+    tokens,
+    activations,
+    task_specific_tag,
+    mappings=None,
+    task_type="classification",
+    binarized_tag=None,
+    balance_data=False,
+    x_dtype=None,
```

Comment:
I think it is clear that the dtype here refers to `X` (since labels would be an np.int array regardless). If you still think the distinction is important, please capitalize X (`X_dtype`)

Reply:
changed to `dtype`, method description specifies that it belongs to X only